webhooks/run_test.go

```diff
@@ -61,13 +61,17 @@ func run(ctx context.Context, cfg *rest.Config, scheme *runtime.Scheme, opts *en
 		},
 	})
 
-	go func(ctx context.Context, backfillComplete chan struct{}) {
-		podMutator.BackfillPermissions(ctx, backfillComplete)
-	}(ctx, backfillComplete)
+	errChan := make(chan error, 1)
+	go func() {
+		if err := podMutator.BackfillPermissions(ctx); err != nil {
+			errChan <- err
+		}
+	}()
+	go func() {
+		if err := mgr.Start(ctx); err != nil {
+			errChan <- err
+		}
+	}()
```

Comment:
If both of these end successfully (`err == nil`) then `run()` will never complete

Reply:
thanks, updated